### PR TITLE
Adds import mode to new app docs

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-6535

Link to docs preview:
https://61522--docspreview.netlify.app/openshift-enterprise/latest/applications/creating_applications/creating-applications-using-cli.html#setting-the-import-mode

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/applications-create-using-cli-modify.adoc [hunk 1]
```diff
@@ -199,3 +199,17 @@ To search for images, templates, and other inputs for the `oc new-app` command,
 ----
 $ oc new-app --search php
 ----
+
+== Setting the import mode 
+
```
Comment (prb112) [inline modules/applications-create-using-cli-modify.adoc:204]: Thanks!

**Comment by flavianmissi** [comment]
LGTM!

File: modules/applications-create-using-cli-modify.adoc [hunk 2]
```diff
@@ -199,3 +199,17 @@ To search for images, templates, and other inputs for the `oc new-app` command,
...
+To set the import mode when using `oc new-app`, add the `--import-mode` flag. This flag can be appended with `Legacy` or `PreserveOriginal`, which provides users the option to trigger builds using a single sub-manifest, or all manifests, respectively.
```
Comment (xiuwang) [inline modules/applications-create-using-cli-modify.adoc:205]: @stevsmit Hi, we said here to trigger build, but the example just to create imagestream and deployment, do we need update this description or the example cmds?

The example cmd only create imagestream and deployment
```oc new-app --image=registry.redhat.io/ubi8/httpd-24:latest  --import-mode=PreserveOriginal --name=test
error: unknown flag: --import-mode
See 'oc new-app --help' for usage.
xiuwang@Xiuwangs-MacBook-Pro 414 % ./oc new-app --image=registry.redhat.io/ubi8/httpd-24:latest  --import-mode=PreserveOriginal --name=test
--> Found container image 13086f1 (5 seconds old) from registry.redhat.io for "registry.redhat.io/ubi8/httpd-24:latest"

    * An image stream tag will be created as "test:latest" that will track this image

--> Creating resources ...
    imagestream.image.openshift.io "test" created
    deployment.apps "test" created
--> Success
    Run 'oc status' to view your app.
```

**Comment by xiuwang** [comment]
LGTM

File: modules/applications-create-using-cli-modify.adoc [hunk 3]
```diff
@@ -199,3 +208,18 @@ To search for images, templates, and other inputs for the `oc new-app` command,
 ----
 $ oc new-app --search php
 ----
+
+[id="setting-the-import-mode"]
+== Setting the import mode 
+
+To set the import mode when using `oc new-app`, add the `--import-mode` flag. This flag can be appended with `Legacy` or `PreserveOriginal`, which provides users the option to create image streams using a single sub-manifest, or all manifests, respectively.
```
Comment (JoeAldinger) [inline modules/applications-create-using-cli-modify.adoc:215]: I would maybe add either `ImportMode` values of before or values after `legacy` or PreserveOriginal`.

**Review by JoeAldinger — COMMENTED** [review]
small suggestion

**Pull request merged**